app/src/main/java/com/mybus/view/FavoriteNameAlertDialog.java

```diff
@@ -0,0 +1,94 @@
+package com.mybus.view;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.EditText;
+
+import com.mybus.R;
+import com.mybus.listener.FavoriteAddListener;
+import com.mybus.listener.FavoriteEditOldListener;
+
+import butterknife.ButterKnife;
+
+/**
+ * Created by Lucas De Lio on 6/13/2016.
+ */
+public class FavoriteNameAlertDialog extends DialogFragment {
+
+    private FavoriteAddListener mOnAddNewFavoriteListener;
+    private FavoriteEditOldListener mOnEditOldFavoriteListener;
+    private String mDialogType;
+    public final static String TYPE_EDIT ="TYPE_EDIT";
```

Comment:
check the modifierOrder:
http://checkstyle.sourceforge.net/config_modifier.html